src/index.js

```diff
@@ -2,16 +2,10 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
     <App />
```

Comment:
- [ ] [**OPTIONAL**] Kindly re-adjust the size of your app design. It matches the given design but I think it too big 👍 

Designs

![Capture](https://github.com/oluwatobi-fasasi/magic-math/assets/80458199/30d2cf6f-48f5-4680-85a5-d08f5fae51b8)

![Capture](https://github.com/oluwatobi-fasasi/magic-math/assets/80458199/114286b6-5be5-4c8a-86a6-50c70646b90f)